docs/machine-learning/resources/tasks.md

```diff
@@ -1,42 +1,52 @@
-# Machine Learning Tasks
-After understanding what you are hoping to achieve with your data, you will need to pick the right machine learning task for your situation. Below, you can find an explanation of different machine learning tasks and some common use cases. 
+---
+title: Machine learning tasks
+description: Explore the different machine learning tasks supported in ML.NET.
+ms.date: 05/30/2018
+---
+# Machine learning tasks
+
+When building a machine learning model, you will first need to define what you are hoping to achieve with your data. After, you can pick the right machine learning task for your situation. The following list will help you understand the different machine learning tasks that you can choose from and some common use cases. 
```

Comment:
nit: *...you will first need...* -> *...you first need...*

Also not use the future tense in the last sentence.